proposals/csharp-12.0/ref-readonly-parameters.md

```diff
@@ -255,6 +274,25 @@ The same warning could be reported when using custom collection initializer or i
 
 #### [Method conversions][method-conversions]
 
+We could allow conversions of `ref` method/lambda parameters to `in`/`ref readonly` delegate parameters.
```

Comment:
Similar comment for this statement. I have no idea what converting parameters mean, I do not think language has a concept like that.